assets/src/libs/ZDClient.js

```diff
@@ -22,7 +22,8 @@ const ZDClient = {
    * Set getters for privite objects
    */
   app: {
-    get settings() { return APP_SETTINGS; }
+    get settings() { return APP_SETTINGS; },
+    get client() { return CLIENT; }
```

Comment:
why should we expose the client outside?

Reply:
we should not